src/index.js

```diff
@@ -64,28 +76,28 @@ const App = () => {
 
         // 1. if exists -->> flip it
         // 2. if dont exists --> destroy everything in it, create this key
-        if (sortConfig['columnKey'] === columnToSort){  
+        if (sortConfig['columnKey'] === columnToSort) {
             direction = sortConfig['dirToSort'] === 'asc' ? 'des' : 'asc';
```

Comment:
This seems redundant.
```suggestion
            direction = sortConfig['dirToSort']
```